src/definitions/collections/message.less

```diff
@@ -173,6 +173,10 @@
   .ui.centered.message,
   .ui.center.aligned.message {
     text-align: center;
+    justify-content: center;
+    & > i.icon + .content {
+      flex: 0 0 auto;
+    }
```

Comment:
I think it's better to omit adjacent sibling selector for `content`. So, we can also place icon at the right side (https://jsfiddle.net/ko2in/8r9axfjo/2/). The icon left margin needs to be defined in this case. Maybe, another PR for the icon's margin adjustments.

Reply:
@ko2in Thanks for the hint! I adjusted it according to your suggestion by 0bdd81fe4614ec38321ed69b2240b3677ac8a261
Please review again 🙂

Reply:
And while the right icon also works now, i added a `right aligned` variant as well to complete it 🙂